README.md

```diff
@@ -48,4 +48,6 @@ Those are all Environment variables needed by the application:
 | ADB2C_POLICY_NAME                        | The Active Directory B2C policy name                                              | string |
 | ADB2C_ADMIN_GROUP_NAME                   | The Active Directory B2C group that identifies admin users                        | string |
 | IN_MEMORY_CACHE_TTL                      | A time to live value for in memory cache that stores user's groups                | number |
-| WALLET_API_URL                           | The PM getWalletV2 API URL                                                        | string |
\ No newline at end of file
+| CSTAR_API_URL                            | The base url of the PM service                                                    | string |
+| CSTAR_API_BASE_PATH                      | Tha base path for the getWalletV2 URL                                             | string |
```

Comment:
The base url of the CStar Apim service
Tha base path for the CStar Apim URL

Reply:
Fixed in 95a58e5